3-scalability/README.md

```diff
@@ -137,17 +162,28 @@ image_registry            = "<your-registry>/mass-ingest"
 image_tag                 = "latest"
 moderne_tenant            = "https://app.moderne.io"  # or your tenant url
 moderne_token             = "arn:aws:secretsmanager:region:account:secret:mass-ingest/moderne-token"
-moderne_publish_url       = "https://artifactory.example.com/artifactory/moderne-ingest/"
 
-# Set either username+password/token or SSH authentication
-# moderne_git_credentials   = "arn:aws:secretsmanager:region:account:secret:mass-ingest/git-credentials"
-# moderne_ssh_credentials   = "arn:aws:secretsmanager:region:account:secret:mass-ingest/ssh-private-key"
+# Storage configuration - choose one option:
+
+# Option A: S3 Storage
+moderne_publish_url       = "s3://your-bucket/path/to/lsts/"
```

Comment:
We don't at the moment support key prefixes, so this example will be confusing to users.

Reply:
good catch, i thought i fixed this everywhere but i did not here